### PR TITLE
DIOS-5077 Give warning level logs their own config not tied to debug

### PR DESCRIPTION
**Pull request by bcostdolby**
…level

### CODE REVIEW

File: include/log.h [hunk 1]
```diff
@@ -177,7 +189,7 @@ inline int Debug(const char *msg, ...)
 
 inline int Warning(const char *msg, ...)
 {
-	if (Logger::IsDebugEnabled())
+	if (Logger::IsWarningEnabled())
```
Comment (harryz2000) [inline include/log.h:192]: It looks we should enable higher level logs when lower level is enabled. Shouldn't we?

Comment (harryz2000) [reply include/log.h:192]: OK. Maybe I was wrong. The levels look independent except the UltraDebug and Debug.

**Review by harryz2000 — APPROVED** [review]
lgtm

**Comment by bcostdolby** [comment]
This is a very small change no changes were required so merging now so I can properly update the other repos to use it (need the submodule hash updated).

**Pull request merged**